rewrite-java/src/main/java/org/openrewrite/java/tree/J.java

```diff
@@ -1053,6 +1056,46 @@ public EnumValueSet getEnumValues() {
             return new FindType(clazz).visit(this);
         }
 
+        public static J.ClassDecl addAnnotation(
+                J.ClassDecl c,
+                boolean isTopLevelClass,
```

Comment:
You should be able to simplify and remove this now. Add the annotation and then apply `AutoFormat`. It's only used to detect a special case of formatting that should be caught in `AutoFormat`.